app/src/main/res/values-es/strings.xml

```diff
@@ -211,8 +222,10 @@
 
     <string name="connectionWizard_providerSelection_titleMessage">Selecciona un proveedor de wallabag</string>
     <string name="connectionWizard_providerSelection_provider_general_name">Otro</string>
-    <string name="connectionWizard_providerSelection_provider_general_desc">Selecciona esta opción para introducir detalles manualmente</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_name" translatable="false">v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_general_desc">Selecciona esta opción para introducir los ajustes requeridos manualmente</string>
     <string name="connectionWizard_providerSelection_provider_v2wallabagorg_desc">Selecciona esta opción si tienes una cuenta en https://v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_name" translatable="false">Framabag.org</string>
```

Comment:
`translatable="false"` strings should be present only in default locale files (in `res/values/` directory).